Gruntfile.js

```diff
@@ -30,9 +35,24 @@ module.exports = function (grunt) {
   // Define the configuration for all the tasks
   grunt.initConfig({
 
+    revision: execSync("git rev-parse --verify HEAD  | xargs echo -n"),
```

Comment:
Looks like `revision` and `gitRev` are redundant now. Just get rid of revision and point cache_control at gitRev.